ruby/lang/security/mass-assignment-protection-disabled.yaml

```diff
@@ -5,7 +5,7 @@ rules:
     Mass assignment protection disabled for '$MODEL'. This could
     permit assignment to sensitive model fields without intention. Instead,
     use 'attr_accessible' for the model or disable mass assigment using
-    'config.active_record.whitelist_attributes = true'.
+    'config.active_record.allowlist_attributes = true'.
```

Comment:
Should this be changed? It looks like it's code so we'll need to make sure this variable name was changed too.

Reply:
this is a configuration status in Ruby, (they still use whitelist/blacklist), so we should keep this 😅

Reply:
Changed back, good catch!